test/e2e/shoot/internal/access_adminkubeconfig.go

```diff
@@ -0,0 +1,47 @@
+// Copyright (c) 2022 SAP SE or an SAP affiliate company. All rights reserved. This file is licensed under the Apache Software License, v. 2 except as noted otherwise in the LICENSE file
```

Comment:
Maybe it's worth moving all of these files to a dedicated `test/e2e/shoot/internal/access` package